### ISSUE TITLE
Missing translation files?

### ISSUE THREAD

**Issue by BaumiCoder** [ls1intum/Artemis#9369]
### Describe the bug

In `src/main/webapp/i18n` some files only in one of the subdirectories.

Only in the `en` subdirectory:
- dragAndDropAssignment.json
- ltiOutcomeUrl.json

Only in the `de` subdirectory:
- participationState.json

### To Reproduce

In your working copy of the Artemis repository
```sh
cd src/main/webapp/i18n
ls en > en.txt
ls de > de.txt
diff en.txt de.txt
```
Output of the `diff` command:
```
34d33
< dragAndDropAssignment.json
82d80
< ltiOutcomeUrl.json
98a97
> participationState.json
```

### Expected behavior

The output of the `diff` command is empty, because everything in Artemis should be available in English and German (see [PR-Template](https://github.com/ls1intum/Artemis/blob/00f38c0ce034fadf6d93d66967073edfcf57856f/.github/PULL_REQUEST_TEMPLATE.md?plain=1#L33)).

If only somebody forgot to delete the files in the other language, a GitHub action, doing my little reproduction steps, could maybe prevent such mistakes getting merged in the future.

### Which version of Artemis are you seeing the problem on?

7.5.3

### What browsers are you seeing the problem on?

Other (specify in "Additional context")

### Additional context

Encounters while working on #9368

**Comment by krusche** [comment]
can we integrate this as GitHub action / additional check that all translations are available in English and German? A python script might be possible here as well.
We could also check for duplicates in translation keys.

Future vision: we find missing keys or non used keys as well

**Comment by MaximilianAnzinger** [comment]
I already tried to solved this and also managed to resolve a lot of the issues. However, I simply ran out of time. Feel free to re-open/reuse: #9206

**Comment by b-fein** [comment]
Resolved with #9471.
